src/gui/mzroll/peakdetectiondialog.cpp

```diff
@@ -82,8 +93,8 @@ void PeakDetectionSettings::updatePeakSettings(string key, string value)
         if(QString(v.typeName()).contains("QGroupBox"))
             v.value<QGroupBox*>()->setChecked(std::stod(value));
 
-        if(QString(v.typeName()).contains("QCheckBox"))
-            v.value<QCheckBox*>()->setChecked(std::stod(value));
+/*        if(QString(v.typeName()).contains("QCheckBox"))
+            v.value<QCheckBox*>()->setChecked(std::stod(value));*/
```

Comment:
Why did we comment this out? With this, checkboxes in peak-detection dialog will not be synced anymore from default settings or when loading settings from emDB files.